docs/UserGuide.md

```diff
@@ -115,32 +116,51 @@ Displays the output of `View` command, which expands each ingredient or recipe c
   e.g. `add ingredient -n <name> -q <quantity>[<units>]` can be used as:
      1. `add ingredient -n chicken -q 2` or as
      2. `add ingredient -n chicken -q 2kg`
+    
+- Items in parentheses `()` should come together.
+
+    e.g. `edit ingredient <index> (-<field flag> <new data>)...` can be used as:
+     1. `edit ingredient 3 -q 3` or as 
+     2. `edit ingredient 3 -q 3 -e 11-03-2022`
```

Comment:
I don't really understand this section. From a user perspective what are items here?

Reply:
It's also mentioned that `Words between < and> are parameters to be supplied by the user.`

Does that mean `<field flag>` is a parameter supplied by user too?